src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleasesController.cs

```diff
@@ -164,12 +164,12 @@ public async Task<ActionResult<List<ReleaseStatusViewModel>>> GetReleaseStatuses
                 .HandleFailuresOrOk();
         }
 
-        [HttpPut("releases/{releaseId}")]
-        public async Task<ActionResult<ReleaseViewModel>> UpdateRelease(ReleaseUpdateViewModel request,
+        [HttpPut("releases/{releaseId}/info")]
```

Comment:
Wouldn't change the endpoint, would keep the semantics of this the same i.e. it updates a release.